stride/physics/iso_elastic/devito.py

```diff
@@ -0,0 +1,398 @@
+
+import devito
+import numpy as np
+import scipy.signal
+
+import mosaic
+from mosaic.utils import camel_case
+
+from ..common.devito import GridDevito, OperatorDevito
+from ..problem_type import ProblemTypeBase
+from .. import boundaries
+
+
+__all__ = ['IsoElasticDevito']
+
+
+@mosaic.tessera
+class IsoElasticDevito(ProblemTypeBase):
+    """
+
```

Comment:
Add some short info about what this class is

Reply:
Correction made